frontend/app/src/components/nft/NftGalleryItem.vue

```diff
@@ -100,12 +113,21 @@ export default defineComponent({
     const imageUrl = computed(() => {
       return item.value.imageUrl ?? require('@/assets/images/placeholder.svg');
     });
-    return { name, imageUrl };
+    const isVideo = videoExtensions.some(
+      extension =>
+        item.value.imageUrl !== null && item.value.imageUrl.includes(extension)
```

Comment:
Can you make this a computed property? 

`isVideo` as it is now is not reactive and won't change along with `item`